data/schema.yaml

```diff
@@ -122,17 +117,17 @@ components:
                   enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
                 model:
                   oneOf:
-                  - $ref: '#/components/schemas/Article/properties/model/properties/blocks'
-                  - $ref: '#/components/schemas/Text'
-                  - $ref: '#/components/schemas/Image'
-                  - $ref: '#/components/schemas/Headline'
+                  - $ref: '#/components/schemas/article/properties/model/properties/blocks'
+                  - $ref: '#/components/schemas/text'
+                  - $ref: '#/components/schemas/image'
+                  - $ref: '#/components/schemas/headline'
                   - $ref: '#/components/schemas/altText'
-                  - $ref: '#/components/schemas/Paragraph'
-                  - $ref: '#/components/schemas/Caption'
-                  - $ref: '#/components/schemas/Video'
+                  - $ref: '#/components/schemas/paragraph'
+                  - $ref: '#/components/schemas/caption'
+                  - $ref: '#/components/schemas/video'
                   - $ref: '#/components/schemas/rawVideo'
                   - $ref: '#/components/schemas/rawImage'
-                  - $ref: '#/components/schemas/Model'
+                  - $ref: '#/components/schemas/model'
```

Comment:
Can `blocks` contain a `model`? That doesn't seem right to me.

Reply:
I agree that this line should be removed from here.

Reply:
This was meant to represent the specific instance where model contains just a string inside it and not a block array. I have changed the name of this  to 'modelWithText' to make it more clear.